src/content/en/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -162,7 +163,7 @@ Your server logic should deliver the appropriate build for each browser.
 
 ### Bundled build
 
-For browsers that don't handle HTTP2 Push, the build process could produce a set
+For browsers that don't handle HTTP2, the build process could produce a set
```

Comment:
I was asking myself the same question, but it was “HTTP2” before :D Happy it fix that